DataProcessing/QuiverCongressDataDownloader.cs

```diff
@@ -217,49 +211,46 @@ public bool Run()
         /// <exception cref="Exception">Failed to get data after exceeding retries</exception>
         private async Task<string> HttpRequester(string url)
         {
-            url = Uri.EscapeDataString(url);
```

Comment:
No need for scaping anymore?

Reply:
The new url has `?version=v2`, which we should not escape, or the rul will be invalid down the line.